app/models/search_facet.rb

```diff
@@ -349,13 +338,22 @@ def update_filter_values!
     end
   end
 
+  # return the correct query string for updating filter values from BQ based on facet type
+  def generate_query_string_by_type
```

Comment:
I would just call this method `generate_bq_query_string` since you're not passing in the type